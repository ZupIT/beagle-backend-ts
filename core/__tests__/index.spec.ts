describe('core', () => {
  it.todo('should create tests!')
})
