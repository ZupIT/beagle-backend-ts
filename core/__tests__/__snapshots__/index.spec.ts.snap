// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Core module should export 1`] = `
Object {
  "BeagleJSX": "object",
  "Component": "function",
  "Operation": "function",
  "RootContext": "function",
  "componentValidation": "object",
  "coreNamespace": "string",
  "createAction": "function",
  "createContext": "function",
  "createContextNode": "function",
  "getGlobalContext": "function",
  "isDynamicExpression": "function",
  "serialize": "function",
  "setFragmentFactory": "function",
  "setupHotReloading": "function",
}
`;
